docs/installing-macos.md

```diff
@@ -79,36 +79,35 @@ This list is incomplete. Feel free to improve it.
 
 -------
 
-**Installing macOS**
-
+### Installing macOS
 
 **Getting Started:**
 
 1. Follow the [Dortania guide](https://dortania.github.io/OpenCore-Install-Guide).
 2. Follow the laptop guide for your CPU generation
-   * Amber Lake Chromebooks should follow the Kaby Lake config.plist setup   
-
+  * Amber Lake Chromebooks should follow the Kaby Lake config.plist setup
+3. As necessary, add the following Kexts and SSDTs to your EFI:
 
-**Chromebook Specific Modifications**
+Firmware Fixes:
+  * [croscorebootpatch](https://github.com/meghan06/croscorebootpatch): Fixes freeze during boot when using coreboot 4.20 and newer
 
-1. [ChromebookPS2](https://github.com/meghan06/ChromebookPS2/): Remaps top row FX keys to their intended purpose for Chromebook keyboards
-2. [croscorebootpatch](https://github.com/meghan06/croscorebootpatch): Fixes freeze on booting with coreboot 4.20 and up.
-3. [crossdxcdisable](https://github.com/meghan06/crossdxcdisable): Disables SD card reader as it is unsupported.
-4. [croshdasdisable](https://github.com/meghan06/croshdasdisable): Disables the `HDAS` device in ACPI as there is no support for it in macOS.
+Keyboard:
+* [VoodooPS2.kext](https://github.com/1Revenger1/VoodooPS2/releases): Fork of Acidanthera's VoodooPS2 which allows mapping keyboard brightness and other useful keys
+* [Keyboard Map](https://github.com/1Revenger1/Acer-Spin-713-Hackintosh/blob/main/src/ACPI/SSDT-ChromeKeys.dsl): Maps FNx keys to ChromeOS mapping. This is an example, you may need to do your own mapping.
```

Comment:
You should add the fact that they will have to compile the SSDT too

Reply:
Done